presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -31,6 +32,8 @@
     private Optional<String> awsAccessKey = Optional.empty();
     private Optional<String> awsSecretKey = Optional.empty();
     private Optional<String> catalogId = Optional.empty();
+    private int numGlueSegments = 5;
+    private int maxGlueGetPartitionThreads = 50;
```

Comment:
- no need for "Glue" here
- add "total" to emphasize this is total cap, not per invocation:

`maxGetPartitionTotalThreads` ?

Also, what is the rationale for 50 as the default?
What's the default request limit for Glue GetPartition call?
What's the typical duration of a call?

Also, should this be off by default?

Reply:
Will change variable names. Typically the call is expected to take hundreds of millis. 5 concurrent segments is conservative enough to be one by default.